internal/backend/httpsvc/intercept.go

```diff
@@ -73,8 +128,10 @@ func intercept(z *zap.Logger, cfg *LoggerConfig, extractors []RequestLogExtracto
 
 		next.ServeHTTP(rwi, r)
 
-		duration := time.Since(startTime)
-		w.Header().Add("X-AutoKitteh-Duration", duration.Truncate(time.Microsecond).String())
+		duration := time.Since(startTime).Truncate(time.Microsecond)
```

Comment:
why truncate?

Reply:
we were reporting it in logs with millisecond resolution (e.g. truncating microseconds). Metrics are also on millisecond resolution

Reply:
doesn't matter - i see no use for truncate here.

Reply:
done